docs/source/resources/frequently_asked_questions.ipynb

```diff
@@ -958,8 +961,8 @@
     "                           \"gender\": [\"f\", \"m\", \"m\", \"m\", \"m\"],\n",
     "                           \"signup_date\": pd.date_range('2014-01-01 01:41:50', periods=5, freq='25min')})\n",
     "\n",
-    "# lets add NaN label column to the test Dataframe\n",
-    "test_train['labels'] = np.nan\n",
+    "# lets remove the labels feature\n",
+    "test_features = [feature for feature in features_enc if feature.get_name() != \"labels\"]\n",
```

Comment:
Something like this?

> Now we can use the feature definitions to calculate our feature matrix for the test data, and avoid label leakage.  Since the test data does not include labels, we will remove the labels feature from the list of encoded features before calculating on the test data.

Reply:
That works, but maybe we say something like:

> Since we do not want the test data to include the labels, let's remove the labels feature from the list of encoded features before calculating on the test data.

Reply:
Thinking about this a bit more, how do we want users to get the labels associated with the test data?

Reply:
Should users include the labels feature, then remove it after for scoring?

Reply:
Yeah we could that, but in that case we should use actual boolean values for the column instead of NaNs.

Normally we would recommend adding the labels to the cutoff time dataframe and they'd be passed through that way, but this is a scenario with no cutoff times

Reply:
Cool, I think it'd be helpful to reflect how users would go about separating the associated labels from the test data.